packages/sdk/lib/node.ts

```diff
@@ -13,6 +13,9 @@ export {
   ValidationOptions,
   ValidationResponse,
   Logger,
+  RequestLoggerPlugin,
```

Comment:
Now that we no longer promote importing browser/node by path and only via `import {ItlyNode} @itly/sdk` we can probably remove all these re-exports from base and only export `Itly` here and in browser.ts.

Reply:
Done